src/views/base_view.js

```diff
@@ -34,6 +37,11 @@ export default AmpersandView.extend({
     dom.hide(this.el);
   },
 
+  // Adjust the height of the iframe to match the height of the body.
+  adjustHeight () {
+    webChannel.sendAdjustHeight(document.body.offsetHeight);
```

Comment:
I'm not sure we need the offsetHeight of the body. Don't we just need the height of the outermost view's container element?

In general, I think we should always aim for View code to not reach outside its top-level container element.